jcl/src/openj9.dataaccess/share/classes/com/ibm/dataaccess/ByteArrayMarshaller.java

```diff
@@ -1,6 +1,6 @@
 /*[INCLUDE-IF DAA]*/
 /*******************************************************************************
- * Copyright (c) 2013, 2015 IBM Corp. and others
+ * Copyright (c) 2013, 2017 IBM Corp. and others
```

Comment:
Pls keep previous changes, i.e., just move the file and not modify it which has better chance to let EGit connect file history. Copyright year changing can be in another pull request.

Reply:
The copyright check fails without updating the copyright. Is that alright?

Reply:
I think it is ok to merge with copyright check failure in this case unless @DanHeidinga says otherwise.

Reply:
If its only a copyright check failure on a file that's been moved but not modified, then I agree with Jason that we're OK to ignore the check.  We haven't modified the code, so we don't need to change the date.